tests/chainer_tests/links_tests/rnn_tests/test_link_n_step_rnn.py

```diff
@@ -413,4 +414,74 @@ def test_n_cells(self):
         assert self.rnn.n_cells == 1
 
 
+@testing.parameterize(
+    *testing.product(
+        {
+            'dtype': [numpy.float32, numpy.float64],
+            'initializer': ['random', None],
+            'activation_type': ['tanh', 'relu'],
+            'use_bi_direction': [True, False]
+        }
+    )
+)
+class TestInitialization(unittest.TestCase):
+    def setUp(self):
+        if self.initializer is None:
+            initializer = initializers.constant.Zero()
```

Comment:
`'zero'` could be more informative than `None`.

Reply:
8cc5033
Thanks, I updated.